tools/data-conversion/src/main.rs

```diff
@@ -60,8 +61,8 @@ struct Arguments {
     #[argh(option)]
     from: String,
 
-    /// file to convery to
-    #[argh(option)]
+    /// optional file to convery to
+    #[argh(option, default = "String::from(\"\")")]
     to: String,
```

Comment:
If you make this an `Option<String>`, then the default (no output file; print to stdout) will be easier to distinguish:
https://doc.rust-lang.org/std/option/

This way, usage will look like this:

* To print to a file, use `--to something.txt`.
* To print to stdout, use nothing at all.

(So there is no need for a separate `-o` option.)